docker/Dockerfile.amd

```diff
@@ -45,6 +45,16 @@ ENV PATH=${PATH}:/opt/openmpi/bin:$AOMP/bin
 ENV LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/opt/openmpi/lib:$AOMP/lib
 ENV OMPI_CC=$AOMP/bin/clang
 
+# Install Omnitrace, an AMDResearch profiling tool which is de facto the best profiler from AMD
+RUN wget https://github.com/AMDResearch/omnitrace/releases/download/v1.7.3/omnitrace-1.7.3-ubuntu-20.04-ROCm-50100-PAPI-OMPT-Python3.sh -q
```

Comment:
Should this be somehow linked to the `rocm` version at the top?

Reply:
Probably, but... it's a bit tedious now

Also the OS right?

What if I leave a comment at the top to remember to update this accordingly if the ROCM_VERSION changes?

Because whatever I do here it still sounds quite fragile regardless. But anyway open to ideas... what did you have in mind? parse the ROCM_VERSION and transform it in bash into "50100" or something like that?

Reply:
I will leave a question in their github

Reply:
Done: https://github.com/AMDResearch/omnitrace/issues/217

Reply:
That's a bad naming convention jeez. Yeah may have to just stick to a given version

Reply:
The naming convention arises from all the optional 3rd party library support. PAPI support is optional, so is OpenMP tools, so it Python, and there's also build variants with full MPI support (it will aggregate profiling data), and you can even build it with support with callbacks to others profilers like TAU, Caliper, etc. There's actually a fair amount excluded at this point: Kokkos-Tools, partial MPI support, and RCCL support are included but I gave up detailing them. I've considered dropping OMPT and PAPI since those do not require runtime link dependencies similar to Kokkos-Tools/partial MPI/RCCL but I've tried to avoid changing the convention

Reply:
Ah, in fact it may be worth noting, ROCm support is optional too. You can use omnitrace on pure CPU code